plugins/cmd/capabilities/log-event-trigger/main.go

```diff
@@ -105,12 +109,16 @@ func (cs *LogEventTriggerGRPCService) Initialise(
 
 	// Set relayer and trigger in LogEventTriggerGRPCService
 	cs.config = logEventConfig
-	cs.trigger, err = logevent.NewTriggerService(ctx, cs.s.Logger, relayer, logEventConfig)
+	triggerService, err := logevent.NewTriggerService(ctx, cs.s.Logger, relayer, logEventConfig)
+	if err != nil {
+		return fmt.Errorf("error creating trigger service for chainID %s: %v", logEventConfig.ChainID, err)
+	}
+	err = triggerService.Start(ctx)
 	if err != nil {
-		return fmt.Errorf("error creating new trigger for chainID %s: %v", logEventConfig.ChainID, err)
+		return fmt.Errorf("error starting trigger service for chainID %s: %v", logEventConfig.ChainID, err)
 	}
 
-	if err := capabilityRegistry.Add(ctx, cs.trigger); err != nil {
+	if err := capabilityRegistry.Add(ctx, cs.triggerService); err != nil {
```

Comment:
This line references the field `cs.triggerService`, but the preceding lines use a local var `triggerService`.

Reply:
Updated cs.triggerService